code/SQLite3Database.php

```diff
@@ -2,25 +2,30 @@
 
 namespace SilverStripe\SQLite;
 
-use Convert;
-use File;
-use SilverStripe\ORM\DataList;
+use SilverStripe\Assets\File;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Convert;
+use SilverStripe\Dev\Deprecation;
 use SilverStripe\ORM\ArrayList;
 use SilverStripe\ORM\Connect\SS_Database;
-use Config;
-use Deprecation;
-use PaginatedList;
+use SilverStripe\ORM\DataList;
 use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\PaginatedList;
 use SilverStripe\ORM\Queries\SQLSelect;
 
-
 /**
  * SQLite database controller class
- *
- * @package SQLite3
  */
 class SQLite3Database extends SS_Database
 {
+    use Configurable;
+
+    /**
+     * Extension added to every database name
+     *
+     * @var string
+     */
+    private static $database_extension = '.sqlite';
```

Comment:
Shouldn't we add an `@config` annotation to any "configurable" private static? At least that what we document in core: https://github.com/silverstripe/silverstripe-framework/blob/ca1716b9f3b8ac7f777ae42e723b9aa464361674/docs/en/02_Developer_Guides/04_Configuration/00_Configuration.md